### DISCUSSION TITLE
psp version 🙏

### DISCUSSION THREAD

**Discussion by ayskura** [jtothebell/fake-08#138] [Q&A] [answered]
Would it be fasible to have a psp version? The psp is still an incredible piece of hardware and should be capable to run pico 8 games

**Comment by jtothebell** [comment, marked as answer]
I think it would be possible to build, but it looks like the psp's cpu is slow enough that a lot of games wouldn't run at full speed, so it isn't near the top of my priority list. I'd welcome and PRs if anyone else wanted to take a stab at it though.
